src/resources/views/base/auth/passwords/reset.blade.php

```diff
@@ -2,8 +2,8 @@
 
 @section('content')
     <div class="row justify-content-center">
-        <div class="col-12 col-md-8 col-lg-5">
-            <h3 class="text-center m-b-20">{{ trans('backpack::base.reset_password') }}</h3>
+        <div class="col-12 col-md-9 col-lg-6">
```

Comment:
Tabs float on top of each other at some localisations, because the translation takes many characters. In the mobile version, we can still neglect this, but not on the desktop.